soc/nordic/nrf53/Kconfig

```diff
@@ -163,10 +163,11 @@ config NRF_SPU_RAM_REGION_SIZE
 	  RAM region size for the NRF_SPU peripheral
 
 config SOC_NRF_GPIO_FORWARDER_FOR_NRF5340
-	bool
+	bool "Forward GPIO pins to network core"
 	depends on NRF_SOC_SECURE_SUPPORTED
+	default y if $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF_GPIO_FORWARDER))
```

Comment:
I think the existing occurrences of `select SOC_NRF_GPIO_FORWARDER_FOR_NRF5340 if ...` should be removed together with this being introduced.

Reply:
Fixed